integration_tests/tests/manage/cancellation.cy.ts

```diff
@@ -138,7 +141,10 @@ context('Cancellation', () => {
     // Given a booking is available
     const application = applicationFactory.build()
     const placement = cas1SpaceBookingFactory.upcoming().build({ applicationId: application.id })
-    const premises = extendedPremisesSummaryFactory.build({ bookings: [placement], id: placement.premises.id })
+    const premises = extendedPremisesSummaryFactory.build({
```

Comment:
I think this can be changed to `const premises = premisesFactory.build({ id: placement.premises.id })` then it doesn't need bookings

Reply:
I've done what you just suggested, seems all types and the related test are all fine. Thanks!